client/core/chrome-detect.js

```diff
@@ -29,6 +31,20 @@
       browserHasExtension = (window.cordova || window.chrome.cast.extensionId || socketDev);
       return browserHasExtension;
     }
+
+    function checkMobile () {
+      // if the user is on mobile, direct them to our site to get the native clients
+      browserIsMobile = ( window.navigator.userAgent.match(/Android/i) ||
+                          window.navigator.userAgent.match(/webOS/i) ||
+                          window.navigator.userAgent.match(/iPhone/i) ||
+                          window.navigator.userAgent.match(/iPad/i) ||
+                          window.navigator.userAgent.match(/iPod/i) ||
+                          window.navigator.userAgent.match(/BlackBerry/i) ||
+                          window.navigator.userAgent.match(/Windows Phone/i) );
+
+      return browserIsMobile;
```

Comment:
Good catch, I'll test that on mobile this morning-

On Wed, Apr 29, 2015, 10:43 PM Andy Kitson <notifications@github.com> wrote:

> In client/core/chrome-detect.js
> <https://github.com/fortunate-giraffe/memes-for-mankind/pull/331#discussion_r29404924>
> :
>
> > @@ -29,6 +31,20 @@
> >        browserHasExtension = (window.cordova || window.chrome.cast.extensionId || socketDev);
> >        return browserHasExtension;
> >      }
> > +
> > +    function checkMobile () {
> > +      // if the user is on mobile, direct them to our site to get the native clients
> > +      browserIsMobile = ( window.navigator.userAgent.match(/Android/i) ||
> > +                          window.navigator.userAgent.match(/webOS/i) ||
> > +                          window.navigator.userAgent.match(/iPhone/i) ||
> > +                          window.navigator.userAgent.match(/iPad/i) ||
> > +                          window.navigator.userAgent.match(/iPod/i) ||
> > +                          window.navigator.userAgent.match(/BlackBerry/i) ||
> > +                          window.navigator.userAgent.match(/Windows Phone/i) );
> > +
> > +      return browserIsMobile;
>
> Think we also need to check !window.cordova here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fortunate-giraffe/memes-for-mankind/pull/331/files#r29404924>
> .
>